src/System.Management.Automation/engine/InitialSessionState.cs

```diff
@@ -4764,7 +4764,8 @@ internal static SessionStateAliasEntry[] BuiltInAliases
 
 # Respect PAGER, use more on Windows, and use less on Linux
 if (Test-Path env:PAGER) {
-    $moreCommand = (Get-Command -CommandType Application $env:PAGER | Select-Object -First 1).Definition
+    $pager,$moreArgs = $env:PAGER -split ' ',2
```

Comment:
If there are multiple arguments, this fix would put quotes around the single value in `$moreArgs`.

For a quick fix, this should probably be `... -split '\s+'`.

By removing the limit, `$moreArgs` will be either a string or array, and passing an array to a native command will put spaces between the args. And you should support multiple spaces between args.

Note that this code is still wrong for quoted arguments. If that mattered, you would use the PowerShell parser to parse the command line and extract arguments from the ast.

Reply:
@lzybkr for now, I'm going with the quick fix which should address the majority of usage.  We can revisit support for quoted args if it impacts someone.